extensions/gdx-setup/src/com/badlogic/gdx/setup/GdxSetupUI.java

```diff
@@ -86,6 +87,7 @@
 	ProjectBuilder builder;
 	List<ProjectType> modules = new ArrayList<ProjectType>();
 	List<Dependency> dependencies = new ArrayList<Dependency>();
+	ArrayList<ExternalExtension> externalExtensions = new ArrayList<ExternalExtension>();
```

Comment:
funny story, I made it List first, then got carried on with other thing, copy pasted a line back, and it became ArrayList again. So close! :D